gf-webapp/src/Index.scss

```diff
@@ -66,4 +66,53 @@ $input-padding: 10px;
 $input-font-size: 18px;
 $input-font-family: Montserrat Regular;
 $input-border: 2px solid $color-primary;
-$input-border-focus: 2px solid $color-secondary;
\ No newline at end of file
+$input-border-focus: 2px solid $color-secondary;
+
+//room titles styles
+$room-title-font-family: Montserrat Regular;
+
+//room h1 titles styles
+$room-h1-display: flex;
+$room-h1-flex-direction: column;
+$room-h1-align-items: center;
+$room-h1-justify-content: center;
+$room-h1-position: absolute;
+$room-h1-top: 0;
+$room-h1-left: 0;
+$room-h1-right: 0;
+$room-h1-bottom: 0;
+$room-h1-color: $color-white;
+$room-h1-font-size: 60px;
+
+@mixin room-h1 {
+  display: $room-h1-display;
+  flex-direction: $room-h1-flex-direction;
+  align-items: $room-h1-align-items;
+  justify-content: $room-h1-justify-content;
+  position: $room-h1-position;
+  top: $room-h1-top;
+  left: $room-h1-left;
+  right: $room-h1-right;
+  bottom: $room-h1-bottom;
+  color: $room-h1-color;
+  font-size: $room-h1-font-size;
+  font-family: $room-title-font-family;
+}
+
+//room h2 titles styles
+$room-h2-color: $color-text;
```

Comment:
Delete duplicate. We already have colors consts

Reply:
Done